etc/init.d/lncm-post

```diff
@@ -29,6 +29,19 @@ start() {
     fi
     /sbin/setup-apkcache /var/cache/apk
   }
+  
+  install_python3() {
+    echo "Install Python 3 and upgrade PIP"
+    /sbin/apk update && \
+    /sbin/apk add python3 python3-dev curl jq
```

Comment:
Perhaps `curl`, and `jq` should be in sth like `install_utils` fn?

Reply:
good idea